### PR TITLE
types updates

### PR DESCRIPTION
**Pull request by kenyon**
Just fixing a couple of omissions that I noticed.

### CODE REVIEW

File: types/actions/outputs/omrelp.pp [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Struct data type for the Rsyslog RELP module's Action options
 type Rsyslog::Actions::Outputs::Omrelp = Struct[{
   target               => Variant[Stdlib::Fqdn, Stdlib::IP::Address],
+  port                 => Optional[Stdlib::Port],
```
Comment (dhollinger) [inline types/actions/outputs/omrelp.pp]: I'm not seeing a `port` option in the Rsyslog omrelp docs: https://www.rsyslog.com/doc/v8-stable/configuration/modules/omrelp.html

File: types/modules/input.pp [hunk 2]
```diff
@@ -6,6 +6,7 @@
   'imkafka',
   'imklog',
   'imkmsg',
+  'immark',
```
Comment (dhollinger) [inline types/modules/input.pp]: Where does `immark` come from? I don't see it in the list in the Rsyslog documentation?

File: types/actions/outputs/omrelp.pp (hunk 1 above)
Comment (kenyon) [reply types/actions/outputs/omrelp.pp]: Yeah, the documentation is incomplete: https://github.com/rsyslog/rsyslog-doc/pull/831

File: types/modules/input.pp (hunk 2 above)
Comment (kenyon) [reply types/modules/input.pp]: https://github.com/rsyslog/rsyslog-doc/pull/830

**Pull request merged**